sdk/servicebus/service-bus/src/core/streamingReceiver.ts

```diff
@@ -80,7 +81,16 @@ export class StreamingReceiver extends MessageReceiver {
     if (!options) options = {};
     if (options.autoComplete == null) options.autoComplete = true;
     const sReceiver = new StreamingReceiver(context, options);
-    await sReceiver._init();
+
+    const config: RetryConfig<void> = {
+      operation: () => {
+        return sReceiver._init();
+      },
```

Comment:
This is the only place w.r.t the retry additions where I cannot test this without messing up with the src code... since there is no way to mock the _init method to throw an error and retry. Hence, cannot add tests by mocking the internal functions as I did for the rest.

Reply:
Manually tested, working fine.

Reply:
I do think that it's okay to modify the code to allow injecting a dependency but I understand your hesitation :)